DNN Platform/Library/Entities/Portals/StyleColor.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Entities.Portals
+{
+    using System;
+    using System.Text.RegularExpressions;
+
+    using DotNetNuke.Abstractions.Portals;
+
+    /// <summary>
+    /// Represents a CSS color and its components.
+    /// </summary>
+    public class StyleColor : IStyleColor

```

Comment:
Maybe this should be a `struct`, since it represents a single value?

Reply:
Good good, done in https://github.com/dnnsoftware/Dnn.Platform/pull/5851/commits/22256682bde0280ee10268af525e3d820d34990c